byron/chain/formal-spec/blockchain-spec.tex

```diff
@@ -1042,9 +1078,9 @@ \subsection{Block body processing}
     \inference
     { \maxblocksize \mapsto \var{b_{max}} \in \var{pps} && \bsize{b} \leq \var{b_{max}} \\
       \var{bh} \leteq \bhead{b} & \var{vk_d} \leteq \bhissuer{\var{bh}} \\
-      \verify{\var{vk_d}}{\serialised{\butxo{b}}}{(\fun{bhUtxoSig}~\var{bh})} &
-      \verify{\var{vk_d}}{\serialised{\bcerts{b}}}{(\fun{bhDlgSig}~\var{bh})} \\
-      \verify{\var{vk_d}}{\serialised{\bupdpayload{b}}}{(\fun{bhUpdSig}~\var{bh})} \\
+      \fun{hash}~(\butxo{b}) = \fun{bhUtxoProof}~\var{bh} &
```

Comment:
The `hash` function isn't given anywhere in the document. How about writing down at least its type and a short description of what it does?

Reply:
Sure.